app/uk/gov/hmrc/helptosave/actors/ThresholdConnectorProxy.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.actors
+
+import akka.pattern.pipe
+import akka.actor.{Actor, Props}
+import uk.gov.hmrc.helptosave.actors.ThresholdConnectorProxy.{GetThresholdValue, GetThresholdValueResponse}
+import uk.gov.hmrc.helptosave.connectors.ThresholdConnector
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ThresholdConnectorProxy(thresholdConnector: ThresholdConnector) extends Actor {
```

Comment:
can we rename this to `UCThresholdActor` ? or something like that

Reply:
the actor is really acting as a proxy to a threshold connector so the name given is more descriptive then UCThresholdActor I think

Reply:
Generally actor classes names end with *Actor - so thought of renaming this to ***Actor - and i see its not just acting as a proxy to `thresholdConnector `  because its doing actor stuff in it

Reply:
I guess `ThresholdConnectorProxyActor` is more descriptive then?

Reply:
hmm, thats fine, i will leave it up to @Jackie1988  :)